core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/SnowflakeExpressionBuilder.scala

```diff
@@ -277,7 +277,7 @@ class SnowflakeExpressionBuilder()
     val condition = ctx.predicate().accept(this)
     val thenBranch = ctx.expr(0).accept(this)
     val elseBranch = ctx.expr(1).accept(this)
-    Iff(condition, thenBranch, elseBranch)
+    ir.If(condition, thenBranch, elseBranch)
```

Comment:
https://docs.databricks.com/en/sql/language-manual/functions/iff.html it's a separate function

Reply:
> This function is a synonym for [if function](https://docs.databricks.com/en/sql/language-manual/functions/if.html).